### DISCUSSION TITLE
Option to change the model

### DISCUSSION THREAD

**Discussion by batidiane** [TheR1D/shell_gpt#148] [Ideas]
First, thanks for this implementation, very handy.

Is there an easy way to change the model the cli is using? If I understand correctly, it is using GPT3.5.

**Comment by transcendr** [comment]
lpurdy01 has an open PR for it #85 -- I have been using his fork on the model branch as a docker instance while the main version is installed as well.  I set up an alias `sgtp4` which uses lpurdy's build in docker, while TheR1D's main version is called via the `sgpt` command.  Let me know if you need further instruction.  The details about building and using it from a docker container are in the README.  Otherwise, you just have to add an alias definition to your shell rc which runs that `docker run` command with the `--model gpt-4` argument after you build it.

**Comment by TheR1D** [comment]
Implemented https://github.com/TheR1D/shell_gpt/pull/151
